packages/modules/create-imodel/src/components/base-imodel/BaseIModel.tsx

```diff
@@ -100,15 +137,26 @@ export function BaseIModelPage(props: BaseIModelProps) {
     return (
       !!imodel.name.length &&
       !isPropertyInvalid(imodel.name) &&
-      !isPropertyInvalid(imodel.description)
+      !isPropertyInvalid(imodel.description) &&
+      (!imodel.extent ||
+        (isPointValid(imodel.extent.northEast) &&
+          isPointValid(imodel.extent.southWest)))
     );
   };
 
   const isDataChanged = () => {
     return (
       imodel.name !== initialIModel?.name ||
       imodel.description !== initialIModel?.description ||
-      isThumbnailChanged
+      isThumbnailChanged ||
+      imodel.extent?.northEast.latitude !==
+        initialIModel?.extent?.northEast.latitude ||
+      imodel.extent?.northEast.longitude !==
+        initialIModel?.extent?.northEast.longitude ||
+      imodel.extent?.southWest.latitude !==
+        initialIModel?.extent?.southWest.latitude ||
+      imodel.extent?.southWest.longitude !==
+        initialIModel?.extent?.southWest.longitude
```

Comment:
Unless I missed it, we are never putting back the provided extent to the imodel.extent object, which means that changes via a provided map control will not update the "isDataChanged", which will not enable the Save button.

Reply:
Good catch! Fixed.